vendor/k8s.io/kubernetes/pkg/volume/glusterfs/glusterfs.go

```diff
@@ -476,9 +484,9 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 		return nil, err
 	}
 	r.provisioningConfig = *cfg
-
 	glog.V(4).Infof("glusterfs: creating volume with configuration %+v", r.provisioningConfig)
-	glusterfs, sizeGB, err := r.CreateVolume()
+	reqGid = gidMin + gidRandomizer.Int63n(gidMax)
```

Comment:
@liggitt sure, I can bring this change in upstream asap, `but the gids here should be able to be shaped by a cluster admin… limited to a range, etc`, Do you think the range or min and max values has to be "mandatory" values ? or is it fine to have an "optional" value and work with defaults if there no mention ?